verifact/forum/schema.py

```diff
@@ -39,3 +45,89 @@ def resolve_questions(root, info):
 
     def resolve_answers(root, info):
         return Answer.objects.all()
+
+
+
+class QuestionCreate(Mutation):
+    class Arguments:
+        text = String()
+        citation_url = String()
+        citation_title = String()
+        citation_image_url = String()
+
+    question = Field(QuestionNode)
+
+    def mutate(
+        self,
+        info,
+        text,
+        citation_url,
+        citation_title="",
+        citation_image_url="",
+    ):
+        val = URLValidator()
+        try:
+            val(citation_url)
+        except ValidationError:
+            raise GraphQLError('citationUrl has invalid format!')
```

Comment:
Nice, this is great. But I think we should write the error message with the expectation that it could be shown to a user. For instance: `The citation url is not a valid url.`. You may also consider putting all errors in a separate file with a namespace in order that we can easily see what errors we've created.

Reply:
Another benefit of abstracting them is that you can reference the same variable for both question and answer citation url errors instead of repeating the string.